core/src/main/java/be/isach/ultracosmetics/util/UtilParticles.java

```diff
@@ -71,34 +71,41 @@ public void run() {
         runnable.runTaskTimer(UltraCosmeticsData.get().getPlugin(), 0, 1);
     }
 
-    public static void display(Particles effect, Location location, int amount, float speed) {
+    public static void drawParticlesWithSpeed(Particles effect, Location location, int amount, float speed) {
         effect.display(0, 0, 0, speed, amount, location, 128);
     }
 
-    public static void display(Particles effect, Location location, int amount) {
+    public static void drawParticles(Particles effect, Location location, int amount) {
         effect.display(0, 0, 0, 0, amount, location, 128);
     }
 
-    public static void display(Particles effect, Location location) {
-        display(effect, location, 1);
+    public static void drawParticle(Particles effect, Location location) {
+        drawParticles(effect, location, 1);
     }
 
-    public static void display(Particles effect, double x, double y, double z, Location location, int amount) {
-        effect.display((float) x, (float) y, (float) z, 0f, amount, location, 128);
+    public static void drawParticlesWithOffset(Particles effect, double offsetX, double offsetY, double offsetZ, Location location, int amount) {
+        effect.display((float) offsetX, (float) offsetY, (float) offsetZ, 0f, amount, location, 128);
     }
 
-    public static void display(Particles effect, int red, int green, int blue, Location location, int amount) {
+    public static void drawColoredParticles(Particles effect, int red, int green, int blue, Location location, int amount) {
         for (int i = 0; i < amount; i++)
             effect.display(new Particles.OrdinaryColor(red, green, blue), location, DEF_RADIUS);
     }
 
-    public static void display(int red, int green, int blue, Location location) {
-        display(Particles.REDSTONE, red, green, blue, location, 1);
+    public static void drawColoredParticlesWithSize(Particles effect, int red, int green, int blue, Location location, int amount, float size) {
+        for (int i = 0; i < amount; i++)
+            effect.display(new Particles.OrdinaryColor(red, green, blue), location, DEF_RADIUS, size);
     }
 
-    public static void display(Particles effect, int red, int green, int blue, Location location) {
-        display(effect, red, green, blue, location, 1);
+    public static void drawColoredParticle(Particles effect, int red, int green, int blue, Location location) {
+        drawColoredParticles(effect, red, green, blue, location, 1);
     }
 
+    public static void drawColoredDust(int red, int green, int blue, Location location) {
```

Comment:
Can you make size an optional parameter which defaults to some value so we can remove `drawColoredDustWithSize()` function?

Reply:
rolling back on this comment because java does not support default parameters